ansible/requirements.yml

```diff
@@ -176,3 +176,7 @@
 - name: openmicroscopy.prometheus-node
   src: https://github.com/openmicroscopy/ansible-role-prometheus-node/archive/0.1.1.tar.gz
   version: 0.1.1
+
+- name: openmicroscopy.prometheus-postgres
+  src: https://github.com/openmicroscopy/ansible-role-prometheus-postgres/archive/0.1.0.tar.gz
```

Comment:
Move to galaxy?

Reply:
It depends how long we need to run this in IDR production before we decide it's good enough for general use, but eventually yes.